demos/OISConsole.cpp

```diff
@@ -1,3 +1,475 @@
+<<<<<<< HEAD
```

Comment:
Shit, I tried to update my copy cause I had pushed it to the wrong repo before. Sorry guys.